src/js/edge/edge_shim.js

```
@@ -311,13 +308,15 @@ var edgeShim = {
     window.RTCPeerConnection.prototype.setLocalDescription =
         function(description) {
       var self = this;
-      var sections = SDPUtils.splitSections(description.sdp);
-      var sessionpart = sections.shift();
+      var sections;
+      var sessionpart;
       if (description.type === 'offer') {
         if (!this._pendingOffer) {
         } else { 
           // VERY limited support for SDP munging. Limited to:
           // * changing the order of codecs
+          sections = SDPUtils.splitSections(description.sdp);
+          sessionpart = sections.shift();
```

[shift()](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/shift) will return `undefined` if length is 0, will this cause issues? I.e. do we need to catch this or do we want it to fail loudly?